Theoretical biology prep

drvinceknight opened the pull request.

drvinceknight committed:
Modify the abstract.

This makes the abstract quite long BUT also adds more
introduction/context. This is in line with https://www.sciencedirect.com/science/article/pii/S0022519317305039

Also fix a section header ref.

drvinceknight committed:
Make IPD consistent.

drvinceknight committed:
Address a number of minor issues noticed.

These were mainly picked up by Reviewer 3 from the Physical Review.

drvinceknight committed:
Remove the word "clearly".

drvinceknight committed:
Add a bit more to the literature review.

drvinceknight committed:
Add note about feasibility of a strategy.

Closes #127

drvinceknight committed:
Change known chi -> theoretic chi.

drvinceknight committed:
Update the cover letter.

marcharper commented on main.tex:
```diff
@@ -38,12 +38,29 @@
+readily applied to the Iterated Prisoner's Dilemma. Agents cooperate, at a
+small cost to themselves, in the hope of obtaining a future benefit. In 2012
+a new type of strategy was introduced: zero-determinant strategies which
+made way for strategies that were provable extortionate.
```
provably

marcharper commented on main.tex:
```diff
@@ -60,11 +77,11 @@
+interactive behaviour, having applications in biology, the study of human social
+behaviour, and many other domains. A standard representation  of the game is
+given in equation~\ref{eqn:theipd}, where the constraints ensure a non
+cooperative equilibrium.
```
non-cooperative

marcharper commented on main.tex:
```diff
@@ -76,15 +93,22 @@
-precise algebraic relationship between the probabilities of cooperation given
-the outcome of the previous round of play and slight alterations to these
-probabilities can cause a strategy to no longer satisfy the necessary equations.
+The non cooperative behaviour is no longer necessary when the game is repeated.
```
non-cooperative

marcharper commented on main.tex:
```diff
@@ -76,15 +93,22 @@
-the outcome of the previous round of play and slight alterations to these
-probabilities can cause a strategy to no longer satisfy the necessary equations.
+The non cooperative behaviour is no longer necessary when the game is repeated.
+An initial major piece of research on this topic was the work
```
Maybe just say "Early work in the field showed that..."

drvinceknight replied:
Yeah this is better.

marcharper commented on main.tex:
```diff
@@ -76,15 +93,22 @@
+The non cooperative behaviour is no longer necessary when the game is repeated.
+An initial major piece of research on this topic was the work
+of~\cite{Axelrod1980, Axelrod1980a} which show that a cooperative behaviour
+which was able to respond to uncooperative actions: Tit For Tat would perform
```
"Tit-For-Tat" perhaps, let's just be sure to be consistent throughout

marcharper commented on main.tex:
```diff
@@ -76,15 +93,22 @@
+which was able to respond to uncooperative actions: Tit For Tat would perform
+strongly. The simplicity of Tit For Tat, which only requires knowledge of the
+opponents previous play lead to most research concentrating on these so called
+Memory 1 strategies.
```
memory-one

marcharper commented on main.tex:
```diff
@@ -76,15 +93,22 @@
+Memory 1 strategies.
+
+One specific type of these strategies are zero-determinant (ZD) strategies which
+were introduced in \cite{Press2012}, extortionate strategies have received
```
Split on the comma. "Of these, extortionate ..."

marcharper commented on main.tex:
```diff
@@ -76,15 +93,22 @@
+opponents previous play lead to most research concentrating on these so called
+Memory 1 strategies.
+
+One specific type of these strategies are zero-determinant (ZD) strategies which
```
A subclass of memory-one strategies known as zero-determinant ...

marcharper reviewed (approved):
Just a few minor comments. Thanks for updating and resubmitting the paper!

drvinceknight commented:
Thanks @marcharper, I'll make those changes and submit early next week :+1:

drvinceknight committed:
Make amendments following @marcharper's review

I have left Tit For Tat as is: this is how it was and I believe we're
consistent throughout.

Nikoleta-v3 commented:
Thank you for your work @drvinceknight 🙏🏻
